attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -47,8 +47,7 @@
   @JsonProperty(value = "tenant_id")
   private String tenantId;
 
-  /** if true, UI will render the attribute in group-by widget */
-  private boolean groupable;
+  private boolean groupable = true;
```

Comment:
Not all attributes are groupable though - we just want strings right now. I don't think we get away from logic here. How do the other fields get their defaults set? The default aggregations for example are set based on the attribute type.